pylint-2.7

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env vpython3
```

Comment:
Can this file and pylint_main.py below go in the `ci/` subdirectory instead of at the top level?